xml/System.Buffers/SequenceReader`1.xml

```diff
@@ -772,12 +780,12 @@
         <Parameter Name="advancePastDelimiter" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="span">To be added.</param>
-        <param name="delimiter">To be added.</param>
-        <param name="delimiterEscape">To be added.</param>
-        <param name="advancePastDelimiter">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="span">The read data, if any.</param>
+        <param name="delimiter">The delimiter to look for.</param>
+        <param name="delimiterEscape">If found prior to <paramref name="delimiter" /> it will skip that occurrence.</param>
+        <param name="advancePastDelimiter"><see langword="true" /> to move past the <paramref name="delimiter" /> if found.</param>
+        <summary>Tries to read everything up to the given <paramref name="delimiter" />, ignoring delimiters that are preceded by <paramref name="delimiterEscape" />.</summary>
+        <returns><see langword="true" /> if the <paramref name="delimiter" /> was found.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the <paramref name="delimiter" /> was found; otherwise, <see langword="false" />.</returns>
```